README.md

```diff
@@ -1,130 +1,144 @@
 
-<a id="general-ml-flow"></a>
-### The General ML Pipeline Flow
+<a id="prerequisites"></a>
+## Prerequisites
 
-The various demos follow some or all of the steps shown in the diagram below:
+To run the MLRun demos, first do the following:
 
-<br><p align="center"><img src="./docs/mlrun-pipeline.png" width="800"/></p><br>
+- Prepare a Kubernetes cluster with preinstalled operators or custom resources (CRDs) for Horovod and/or Nuclio, depending on the demos that you wish to run.
+- Install an MLRun service on your cluster.
+  See the instructions in the [MLRun documentation](https://github.com/mlrun/mlrun/blob/master/README.md#installation).
+- Ensure that your cluster has a shared file or object storage for storing the data (artifacts).
 
 <a id="demo-sklearn-pipe"></a>
-## [End-to-End data prep + scikit-learn Pipeline with AutoML (Iris Data Set)](./sklearn-pipe/sklearn-project.ipynb)
+## scikit-learn Pipeline Demo &mdash; AutoML Pipeline (Iris Data Set)
 
-Demonstrate a popular machine learning use case (iris data set), how to explore the data and build an end to end automated ML pipeline.
+The [**sklearn-pipe**](./sklearn-pipe/README.md) demo demonstrates how to build a full end-to-end automated-ML (AutoML) pipeline using [scikit-learn](https://scikit-learn.org) and the UCI [Iris data set](http://archive.ics.uci.edu/ml/datasets/iris).
 
-The combined CI/Data/ML pipeline includes the following steps:
+The combined CI/data/ML pipeline includes the following steps:
 
-- Build the iris generator (ingest) function container.
-- Ingest the iris data.
+- Create an Iris data-set generator (ingestion) function.
+- Ingest the Iris data set.
 - Analyze the data-set features.
 - Train and test the model using multiple algorithms (AutoML).
 - Deploy the model as a real-time serverless function.
-- Test the serverless function REST API with a test data set.
+- Test the serverless function's REST API with a test data set.
 
-To start, download the notebook [sklearn-project.ipynb](./sklearn-pipe/sklearn-project.ipynb) into an empty directory and run the cells one by one.
+To run the demo, download the [**sklearn-project.ipynb**](./sklearn-pipe/sklearn-project.ipynb) notebook into an empty directory and execute the cells sequentially.
 
 <br><p align="center"><img src="./docs/trees.png" width="500"/></p><br>
 
 <a id="demo-sklearn-pipe-pipeline-output"></a>
-#### Pipeline Output
+### Pipeline Output
 
-You can see various output plots in [sklearn-pipe/plots](sklearn-pipe/plots) (static HTML files).
+The output plots can be viewed as static HTML files in the [sklearn-pipe/plots](sklearn-pipe/plots) directory.
 
 <br><p align="center"><img src="./docs/skpipe.png" width="500"/></p><br>
 
 <a id="demo-horovd-image-classification"></a>
-## [Image Classification Using Distributed Training (Horovod)](horovod-pipe/horovod-project.ipynb)
+## Horovod Image-Classification Demo &mdash; Distributed Training
```

Comment:
@Sharon-iguazio no need to change the title, Horvod is a technology (i.e. using Horovod) vs the demo description, u also lost the notebook link

Reply:
Regarding the "lost" NB link, as I explained in [my response](https://github.com/mlrun/demos/pull/40#discussion_r440000170) to another similar comment, I didn't lose the link, I replaced the title NB link with a link to a dedicated demo README (which I added for the Horovod demo) at the beginning of the demo description in this section.